src/neo4j_graphrag/llm/base.py

```diff
@@ -17,7 +17,7 @@
 from abc import ABC, abstractmethod
 from typing import Any, Optional
 
-from .types import LLMResponse
+from .types import LLMMessage, LLMResponse
```

Comment:
Do we need to add the LLMessage type to the doc? We have a `types` page for the types that are user-facing.

Reply:
I have built the docs 👍

Reply:
... and added the type, [7557b07](https://github.com/neo4j/neo4j-graphrag-python/pull/234/commits/7557b07581f8ba1371c4727b3a36548413f53643).